### PR TITLE
dm: fix chaos test

### PR DESCRIPTION
**Pull request by GMHDBJD**
### What problem does this PR solve?

Issue Number: close #4287 

#### Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

### Release note 

```release-note
Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

If you don't think this PR needs a release note then fill it with `None`.
```

### CODE REVIEW

File: dm/chaos/cases/cases.go [hunk 1]
```diff
@@ -27,7 +27,7 @@ var (
 	// NOTE: items in `doSchemas` should be specified in the corresponding task files (`filenames`).
 	// TODO: can not support optimistic shard DDL now because go-sqlsmith will generated some statements like
 	//   `ALTER TABLE `db_optimistic`.`dxeyydwb` ADD COLUMN (`zuhxfgsce` INT(18) NOT NULL)` which has NOT NULL column without DEFAULT value.
-	filenames = []string{"task-single.yaml", "task-pessimistic.yaml", "task-optimistic.yaml"}
+	filenames = []string{"task-single.yaml"}
```
Comment (lance6716) [inline dm/chaos/cases/cases.go:30]: pass it by command line argument?

**Review by lance6716 — APPROVED** [review]

File: dm/chaos/cases/cases.go (hunk 1 above)
Comment (GMHDBJD) [reply dm/chaos/cases/cases.go:30]: Yes, will update it in next pr

**Review by D3Hunter — APPROVED** [review]

**Pull request merged**